tree/20_cfe_basics/cfengine/packages.cf

```diff
@@ -99,6 +99,19 @@ bundle common rpm_knowledge
       "rpm3_arch_regex" string => "\S+\s+(\S+).*";
 }
 
+bundle common redhat_no_locking_knowledge {
+  vars:
+    !redhat_setopt_not_available::
```

Comment:
classes in bundle common are global - so i'm not super happy to add extra classes there that will leak in global context and might overlap